src/components/Button.spec.js

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "@testing-library/react";
+import Button from "./Button";
+
+describe("Button", () => {
+  it("has text and shows the way back", () => {
+    render(<Button children="Back" />);
+    expect(screen.getByRole("button")).toBeInTheDocument();
+  });
+});
```

Comment:
You should also test the isActive and onClick, just when the proptypes is "required".